app/task/project/saveotio/saveotio.cpp

```diff
@@ -135,12 +135,33 @@ OTIO::Track *SaveOTIOTask::SerializeTrack(Track *track)
     if (dynamic_cast<ClipBlock*>(block)) {
       auto otio_clip = new OTIO::Clip(block->GetLabel().toStdString());
 
+      auto rate = 0.0;
+      rational frame_rate = static_cast<ClipBlock*>(block)->connected_viewer()->GetVideoParams().frame_rate();
+      if (frame_rate.denominator() == 1) {
+        rate = frame_rate.numerator();
+      } else {
+        // Handle strange rates
+        rate = static_cast<double>(frame_rate.numerator()) /
+               static_cast<double>(frame_rate.denominator());
```

Comment:
Should the denominator be checked for 0 for dividing?

Reply:
It might be preferable to use `rational::toDouble()` which does effectively the same thing, but does check for a 0 denominator at which point it returns NaN.

Reply:
I think I've addressed the issue here, thanks for the review.